rimage/transform/pinhole_camera_parameters_test.go

```diff
@@ -204,3 +206,17 @@ func TestGetCameraMatrix(t *testing.T) {
 	test.That(t, intrinsicsK.At(1, 2), test.ShouldEqual, intrinsics.Ppy)
 	test.That(t, intrinsicsK.At(2, 2), test.ShouldEqual, 1)
 }
+
+func TestNilIntrinsics(t *testing.T) {
```

Comment:
If a user gets back `nil` intrinsic parameters (which is now possible thanks to my fix 😞) and attempts to call any of the methods below we'd panic. For example

```
props, err := cam.Properties(context.Background())
if err != nil {
	logger.Fatal("attempting to get cam properties: %v", err)
	return
}
props.GetCameraMatrix() // panic!
```

This is easily fixable by checking if the parameters are nil in these methods. It's also idiomatic to do this in Go AFAIK.

I noticed for the distortion parameters we have a struct `transform.NoDistortion` that solves the same problem but I'm not sure what the advantage would be if we adopted that solution for intrinsic parameters too. Seems like they're both valid but I could be missing something. @bhaney it looks like you created that struct so if you have an opinion on this let me know.

Reply:
Yea, I think using nil as the "no intrinsics" and "no distortion" scenario is much more intuitive than creating a specific struct. Because then you have to remember to always use the "no such thing" struct, and nil is still there potentially causing problems!